### PR TITLE
The Vela Class Hazardous Extraction Ship, a GOLD aligned mining ship.

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
The Vela-Class Hazardous Extraction Vessel is a mining hull currently in use by the Galactic Optimum Labor Division for orbital mining operations in the frontier, designed to leverage mechs to rapidly mine out rich ore sites, and then return to offload their ore. The Vessel's interior is highly malleable, although is currently configured in a more unofficial set up, with the fore being dedicated to the crew, and the aft including both a small science lab and vault.. This specific example hasn't seen the best of upkeep practices, but is still a massive vessel capable of projecting soft power into any sector it's deployed in. 

Other examples of this hull exchange the hangar pod for something else, such as troop transports, manufacturing plants, gas harvesting machines, or external cargo bays. Once a Vela is out of dock it is unlikely to stay stock for long.

This Vela has a crew compliment of 19 crew, divided thus:
1 Captain
1 Supply Officer
1 GOLD Delegate
2 Flight Controllers, for coordinating operations of the miners.
4 Pilots
1 Mechanic
2 Engineers
2 Minutemen, assigned to guard the vessel
and 5 cadets, officially doing whatever.

Power is supplied by a quad-Pacman assembly, feeding two SMESes. The air harvester on the starboard could allow for on the go refueling, and allows the ship to endure sustained operations or resupply its atmos supply in the event of catastrophic damage. 
A small shield lattice prevents collisions from and asteroids and debris, further servicing the ship's asteroid harvesting nature.
Near the front is an office installed for the GOLD Delegate, with a built in privacy shutter, secretarial desk, and fax machine for contacting home. 
The cargo bay contains the crew's latest supply of food, several pizzas and a few boxes of donk pockets. It may not be up to some people's standards, but the pilots seem to enjoy it. In the event of the crew finding something edible, a kitchen has been erected in the fore hallway. 
Mech 4 is currently under construction by the Mechanic, once it's ready, space will have to be made in one of the bays for it.

dreamseeker_1QTUmRyT4h
dreamseeker_AAMnigf2tb
dreamseeker_3p5OSOC8Ao
dreamseeker_Vk33GkAywz
dreamseeker_3X72hJXKVB

- [x] Ship lands, ship flies, ship doesn't experience sudden mapper induced decompression, outfits look decent, config is properly set up.
## Why It's Good For The Game
Expanding the minutemen lineup with an something that is not explicitly a warship and or security themed vessel gives more character to the faction. It also adds a physical presence to GOLD, which seems to be kinda important out in the frontier. 

## Changelog

:cl:
add: A Vela-Class hull has been spotted in the region.
add: Vela config
/:cl:

### CODE REVIEW

**Comment by ZephyrTFA** [comment]
Space Pods should not be used in ships due to them being buggy pieces of shit. They are likely going to be removed eventually, unless someone can make them not lag incarnate and also not garbage code

**Comment by Valorium-RB** [comment]
As far as GOLD lore and design is concerned, this gets my Seal of Approval as the GOLD Lore Head. Good work.

**Comment by Erikafox** [comment]
> Space Pods should not be used in ships due to them being buggy pieces of shit. They are likely going to be removed eventually, unless someone can make them not lag incarnate and also not garbage code

My master plan goes something like
>make cool ships that use pods
>someone with actual coding ability notices that pods are janky
>pods get fixed

I'd try it myself honestly but I'm still futzing around with subtyping and outfit defs. The idea of pods is really cool and it'd be a shame to see them gone because they didn't get used and then got removed for being incredibly bad.

**Comment by Erikafox** [comment]
Changed the ship to use mechs, screenshots and everything have been updated.

File: code/modules/spacepods/construction.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/obj/spacepod/examine(mob/user)
+/obj/spacepod/examine(mob/user) //these output above the spacepod's text block for some reason. Not good.
```
Comment (Latentish) [inline code/modules/spacepods/construction.dm]: A. Spacepods 😔 
B. Is this with your spacepod specifically or in general. It seems like the latter.

Comment (Latentish) [reply code/modules/spacepods/construction.dm]: If so, I'd recommend removing this as a change and making a bug report and request the "reproduceable" tag.

**Comment by Latentish** [comment]
image
image
Area fucky wuckies.

File: code/modules/spacepods/construction.dm (hunk 1 above)
Comment (Erikafox) [reply code/modules/spacepods/construction.dm]: I must've missed that when I was unpodding the ship. Gonna remove this.

**Comment by Erikafox** [comment]
I redid the entire interior of the ship because I didn't like the aesthetic of the last version. Screenies and the desc were updated.

**Comment by Erikafox** [comment]
BRANCH NAME CURSED

**Pull request closed without merging**